native/resvg/src/lib.rs

```
@@ -479,42 +533,30 @@ fn parse_options(in_svg: InputFrom, options: Options) -> Result<ParsedOptions, S
     })
 }
 
-fn load_fonts(parsed_options: &ParsedOptions) -> Result<fontdb::Database, String> {
-    let mut fontdb = fontdb::Database::new();
-    if !parsed_options.skip_system_fonts {
+fn load_fonts(font_properties: &FontProperties, fontdb: &mut fontdb::Database)
+        -> Result<(), String> {
+    if !font_properties.skip_system_fonts {
         fontdb.load_system_fonts();
     }
 
-    for path in &parsed_options.font_files {
+    for path in &font_properties.font_files {
+        // FIXME: Ignore failure to load files, for now
```

What happen before the failure was not handled ?